README.md

```diff
@@ -65,10 +66,15 @@ import hipscat as hc
 catalog = hc.Catalog('gaia', location='/path/to/hips/catalog/outputdir')
 ```
 
-The repository will look for an already partitioned catalog at that pathway `location + /output/catname`. If it doesn't find one, it will notify the user to import an existing source catalog utilizing `hipscat.Catalog.hips_import(...)`
+The repository will look for an already partitioned catalog at that pathway `location + /output/catname`. Currently the `location` keyword supports finding hipscat objects on
+* local disk
+* azure blob file system (abfs)
+* aws s3 bucket (s3 boto) coming soon!
 
-This function requires the following parameters:
-* `file_source`: this can be a directory pathway to a existing local catalog or it can be an http api-endpoing serving a list of catalog files.
+If it doesn't find one, it will notify the user to import an existing source catalog utilizing the `hipscat.Partitioner` class.
```

Comment:
I'm going to rip out the `hipscat.Partitioner` stuff, since it's been replaced by `hipscat-import`. I'd suggest pointing folks there instead (even though they're not compatible directory structures).

Reply:
prototype partitioner still exists, but the src/hipscat/partitioner has been ripped out.